localized/games/eggquiz/eggquizmath_levels_ur.tsv

```diff
@@ -388,37 +388,37 @@ SpecialTest_21	1	210001, 210002, 210003, 210003, 210004
 SpecialTest_22	1	220001~220010, 220001~220010, 220001~220010, 220001~220010, 220001~220010
-			220001	word_problem	ملی كے پاس 9 کنگن ہیں . وہ اس میں سے 2 کھو دیتی ہے . ملی كے پاس اب کتنے کنگن بچے ہیں ؟	मिल्ली के पास 9 कंगन हैं। वह उनमें से 2 खो देती है। मिल्ली के पास अब कितने कंगन बचे हैं?				7
+			220001	word_problem	مینا ز كے پاس 9 کنگن ہیں . وہ اس میں سے 2 کھو دیتی ہے . مینا ز كے پاس اب کتنے کنگن بچے ہیں ؟	मिल्ली के पास 9 कंगन हैं। वह उनमें से 2 खो देती है। मिल्ली के पास अब कितने कंगन बचे हैं?				7
 			220002	word_problem	حنا 8 روٹیاں بناتی ہیں . وہ اپنے بھائی کو 3 روٹیاں دیتی ہے . اب اسکے پاس کتنی روٹیاں بچی ہیں ؟	हैना 8 रोटियाँ बनाती है। वह अपने भाई को 3 रोटियाँ देती है। उसके पास कितनी रोटियाँ बची हैं?				5
 			220003	word_problem	آسمان میں 9 پاتانجی اُڑ رہی ہیں . اک ہوا کا جھکا 3 پاتانجون کو جمین پہ گرا دیتا ہے . اب آسمان میں کتنی پاتینجی اُڑ رہی ہیں ؟	आसमान में 9 पतंगें उड़ रही हैं। एक हवा 3 पतंगों को जमीन पर गिरा देती है। कितनी पतंगें अभी भी आसमान में उड़ रही हैं?				6
 			220004	word_problem	ایک طالاب میں 15 بظخ تیر رہی ہیں. 7 بظخ کی پرواز ہوئی. کتنے بظخ طالاب میں رہی ہیں ؟	एक तालाब में 15 बतख तैर रही हैं। 7 बतख उड़ जाती हैं। तालाब में अभी कितने बतख तैर रहे हैं?				8
-			220005	word_problem	لینا کے پاس 12 کتابیں ہیں ۔ وہ اس کی کلاس کی لائبریری میں عطیہ کرتی ہے 8 کتابیں. کتنی کتابیں لینا کے پاس ہیں ؟	लीना के पास 12 किताबें हैं। वह अपनी कक्षा की लाइब्रेरी में 8 किताबें दान करती हैं। लीना के पास अब कितनी किताबें बची हैं?				4
-			220006	word_problem	ایک باڑ پر 12 بلیاں ہیں. 1 بلی باڑ کے نیچے چھلانگ لگاتی ہے. کتنی بلیوں کو باڑ پر چھوڑ دیا جاتا ہے ؟	एक बाड़ पर 12 बिल्लियाँ हैं। 1 बिल्ली बाड़ से नीचे कूदती है। अब कितनी बिल्लियाँ बाड़ पर हैं?				11
-			220007	word_problem	لیری كے پاس 8 پنسل ہے. وہ ان میں سے 7 کھو دیتا ہے .لیری كے پاس کتنی پنسل بچی ہے؟	लूसी के पास 8 पेंसिल हैं। वह उनमें से 7 को खो देती है। लूसी के पास अब कितनी पेंसिल बची हैं?				1
-			220008	word_problem	انا كے پاس 7 بسکٹ ہیں . وہ اس میں سے 2 کھا لیتی ہے. اب اسکے پاس کتنے بسکٹ بچے ہیں؟	एना के पास 7 बिस्कुट हैं। वह उनमें से 2 खाती है। उसके पास कितनी बिस्कुट बचे हैं?				5
+			220005	word_problem	لبنا کے پاس 12 کتابیں ہیں ۔ وہ اس کی کلاس کی لائبریری میں عطیہ کرتی ہے 8 کتابیں. کتنی کتابیں لبنا کے پاس ہیں ؟	लीना के पास 12 किताबें हैं। वह अपनी कक्षा की लाइब्रेरी में 8 किताबें दान करती हैं। लीना के पास अब कितनी किताबें बची हैं?				4
+			220006	word_problem	ایک باڑ پر 12 بلیاں ہیں. 1 بختیار باڑ کے نیچے چھلانگ لگاتی ہے. کتنی بلیوں کو باڑ پر چھوڑ دیا جاتا ہے ؟	एक बाड़ पर 12 बिल्लियाँ हैं। 1 बिल्ली बाड़ से नीचे कूदती है। अब कितनी बिल्लियाँ बाड़ पर हैं?				11
+			220007	word_problem	لیری كے پاس 8 پنسل ہے. وہ ان میں سے 7 کھو دیتا ہے .لیری كے پاس کتنی پنسل بچی ہے؟	आसमा के पास 8 पेंसिल हैं। वह उनमें से 7 को खो देती है। आसमा के पास अब कितनी पेंसिल बची हैं?				1
+			220008	word_problem	انم كے پاس 7 بسکٹ ہیں . وہ اس میں سے 2 کھا لیتی ہے. اب اسکے پاس کتنے بسکٹ بچے ہیں؟	एना के पास 7 बिस्कुट हैं। वह उनमें से 2 खाती है। उसके पास कितनी बिस्कुट बचे हैं?				5
```

Comment:
Properly check Urdu and Hindi text. Either Hindi or Urdu text is not updated at many places